src/System.Threading.Overlapped/tests/ThreadPoolBoundHandle_IntegrationTests.cs

```diff
@@ -15,7 +16,7 @@ public unsafe void SingleOperationOverSingleHandle()
     {
         const int DATA_SIZE = 2;
 
-        SafeHandle handle = HandleFactory.CreateAsyncFileHandleForWrite(@"SingleOverlappedOverSingleHandle.tmp");
+        SafeHandle handle = HandleFactory.CreateAsyncFileHandleForWrite(Path.Combine(TestDirectory, @"SingleOverlappedOverSingleHandle.tmp"));
```

Comment:
Does the actual file name matter for the test, or could GetTestFilePath() be used here, too?